app/controllers/devmode_controller.rb

```diff
@@ -408,7 +408,6 @@ def create_work(name, authors, ratings, warnings, characters, relationships, fan
       :authors => authors,
       :summary => summary,
       :fandoms => fandoms,
-      :ratings => ratings,
```

Comment:
Just for nitpicking's sake, I'd actually leave this one and remove the one on line 419 instead -- this one is more logically placed (it's with the other tag types).